src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -1771,6 +1766,22 @@ protected internal virtual TypeDefinition MarkType (TypeReference reference, Dep
 			MarkCustomAttributes (type, new DependencyInfo (DependencyKind.CustomAttribute, type));
 			MarkSecurityDeclarations (type, new DependencyInfo (DependencyKind.CustomAttribute, type));
 
+			if (type.ToString ().Contains ("Requires"))
+				Debugger.Break ();
+
```

Comment:
Debugging left-over :-)

Reply:
Let's say that is because it was a partial commit xD